lib/LinkEmbedder/Link.pm

```diff
@@ -44,6 +44,7 @@ has ua               => undef;
 has url              => sub { Mojo::URL->new };                                            # Mojo::URL
 has version          => '1.0';
 has width            => sub { $_[0]->type =~ /^photo|video$/ ? 0 : undef };
+has mimetype         => '';
```

Comment:
Can you sort the attribute names alphabetically? Meaning moving this line up a couple of lines. The attribute also needs a "=head2 mimetype" in the POD section.

Reply:
Ok